config/routes.rb

```diff
@@ -23,6 +23,7 @@
 
   resource :preferences, only: [:edit, :update]
   resource :messageboard, path: 'messageboards', only: [:new]
+  resources :messageboard_groups, only: [:new, :create]
```

Comment:
Let's scope all admin routes under `scope path: 'admin' do ... end` and add `admin` to [reserved words](https://github.com/thredded/thredded/blob/master/app/models/thredded/messageboard.rb#L8) to avoid a possible conflict with a messageboard slug. Also the `admin` routes need to be moved above (e.g. line 16) to be matched before the messageboard route.